include.js

```diff
@@ -6,6 +6,13 @@ let cards = [
     author: 'vikasverma67',
     githubLink: 'https://github.com/vikasverma67'
   },
+  {
+    artName: 'ZigZag', 
+    pageLink: './Art/<kendrickche>/ZigZag.html',
+    imageLink: './Art/<kendrickche>/ZigZag.gif', 
+    author: 'Kendrick', 
+    githubLink: 'https://github.com/<kendrickche>'
+  },
```

Comment:
# Chevrons in includes.js 
Chevrons were depicted in the README as placeholder for your real values 
Please remove all chevrons - this create invalid paths and github links
Thanks